packages/booter-lb3app/README.md

```diff
@@ -0,0 +1,48 @@
+# @loopback/booter-lb3app
+
+Boot a LoopBack 3 application and expose its REST API via LoopBack 4.
+
+## Overview
+
+The `@loopback/booter-lb3app` package provides a way for LoopBack 3 developers
+to boot their LoopBack 3 application, convert the application's Swagger spec
+into OpenAPI v3, and then mount the application, including its spec, onto a
+target LoopBack 4 application.
+
+## Installation
+
+```sh
+npm install --save @loopback/booter-lb3app
+```
+
+## Basic use
+
+Import the booter at the top of your `src/application.ts` file.
+
+```ts
+import {Lb3AppBooter} from '@loopback/booter-lb3app';
+```
+
+Register the booter in Application's constructor:
+
+```ts
+this.booters(Lb3AppBooter);
```

Comment:
Would it be better to expose a component from `booter-lb3app` and have the application to mount it?
```ts
import {LB3AppBooterComponent} from '@loopback/booter-lb3app';
this.component(LB3AppBooterComponent);
```

Reply:
That's a good idea to encourage the pattern of extensions packaging artifacts as part of components.